pyproject.toml

```diff
@@ -73,6 +73,18 @@ follow_imports = "skip"
 follow_imports_for_stubs = true
 
 
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+# PYTEST CONFIGURATION                                                        #
```

Comment:
Are the changes to this file related to switching to src layout or is this something separate?

Reply:
it's separate, but would be good to have. VSCode would automatically detect the tests